src/scheduler/MpiWorld.cpp

```diff
@@ -1005,14 +1004,15 @@ void MpiWorld::allToAll(int rank,
 
 void MpiWorld::probe(int sendRank, int recvRank, MPI_Status* status)
 {
+    // TODO - probe is now broken as we need to re-implement queue's peek
```

Comment:
If this TODO is still true we should throw an exception and delete/ comment out the broken implementation

Reply:
I forgot to delete this comment, thanks